docs/ONNX_Runtime_Format_Model_Usage.md

```diff
@@ -0,0 +1,107 @@
+
+Alternatively a configuration file can be created for the set of kernels to include. 
+
+See the documentation on the [Reduced Operator Kernel build](Reduced_Operator_Kernel_build.md) for more information. This step can be run separately or as part of the minimal build.
+
+### Create the minimal build
+
+You will need to build ORT from source to reduce the operator kernels and other aspects of the binary. 
+
+See [here](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#start-baseline-cpu) for build instructions. 
+
+Binary size reduction options.
+  - Enable minimal build (`--minimal_build`)
+    - A minimal build will ONLY support loading and executing ORT format models.
+
+  - Disable RTTI (`--disable_rtti`)
+    - Disabling RTTI support will result in significant savings at the cost of a small number of unlikely error messages losing type information. If you encounter such errors you can attempt to run the model with a full ORT build to get the full error information.
+
+  - Disable exceptions (`--disable_exceptions`)
+    - Disables exceptions in the build. Any locations that would have thrown an exception will instead log the error message and call abort(). 
+        - Requires `--minimal_build`
+        - Is not a valid option if you need the python bindings (`--build_wheel`) as python/pybind cannot be built with exceptions disabled.
+    - Exceptions are only used in ORT for exceptional things. If you have validated the input to be used, and validated that the model can be loaded, it is unlikely that ORT would throw an exception unless there's a system level issue (e.g. out of memory). 
+
+  - ML op support (`--disable_ml_ops`)
+    - Whilst the operator kernel reduction script will disable all unused ML operator kernels, additional savings can be achieved by removing support for ML specific types. If you know your model has no ML ops, or no ML ops that use the Map type, this flag can be provided. 
+    - See the specs for the [ONNX ML Operators](https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md) if unsure.
+
+
+#### Example build commands
+
+##### Windows
+
+`<ORT repository root>\build.bat --config=MinSizeRel --cmake_generator="Visual Studio 16 2019" --build_shared_lib --minimal_build --disable_rtti --disable_ml_ops --disable_exceptions`
+
+##### Linux
+
+`<ORT repository root>/build.sh --config=MinSizeRel --build_shared_lib --minimal_build --disable_rtti --disable_ml_ops --disable_exceptions`
+
+## Executing ORT format models
+
+The API for executing ORT format models is the same as for ONNX models. See the [ORT API documentation](https://github.com/Microsoft/onnxruntime/#api-documentation).
+
+If you provide a filename for the ORT format model, a file extension of '.ort' will be inferred to be an ORT format model.
```

Comment:
is it _.optimized.onnx_ or _.ort_?

Reply:
.ort. There are two steps in model conversion. ONNX -> ONNX + ORT optimizations (.optimized.onnx) -> ORT format (.ort)